client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -32,15 +34,39 @@ const getListHeaderTextForAccountType = ( accountType, translate ) => {
 	} );
 };
 
-const MailboxListHeader = ( { accountType = null, children, isPlaceholder = false } ) => {
+const MailboxListHeader = ( {
+	accountType = null,
+	children,
+	isPlaceholder = false,
+	addMailboxPath,
+	domain,
+	showIcon,
+} ) => {
 	const translate = useTranslate();
 
+	const HeaderIcon = () => (
+		<div className="email-plan-mailboxes-list__mailbox-header-icon">
+			<EmailTypeIcon domain={ domain } />
```

Comment:
```suggestion
			<EmailTypeIcon domain={ domain } size={ 24 } />
```
Nit, but passing this `size` prop to `Gridicon` affects the placement of the WordPress icon and improves the alignment. Would be a nice final touch.